src/sass/components/_tour.scss

```diff
@@ -35,6 +35,15 @@
   background-image: url('~img/popover-header.png');
 }
 
+.popover-reference {
+  position: relative;
+}
```

Comment:
I wonder if it would be clearer to use `.slds-is-relative` throughout, instead of our own custom class.